rusk/src/bin/main.rs

```diff
@@ -75,7 +75,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let router = {
         let rusk = Rusk::builder(disk_backend)
-            .path(rusk_profile::get_rusk_state_id_path()?)
+            .generator(config.wallet.generator())
```

Comment:
You forgot to re-add the `path`. That would break the binary – you won't catch this in the tests since we're not testing yet the CLI.